grid/cell.py

```diff
@@ -152,18 +153,32 @@ def from_id(cls, id: int) -> "Position":
     def id(self) -> int:
         return triangular_number(self.row) + self.col
 
-    @property
-    def x(self) -> int:
-        return self.col
-
-    @property
-    def y(self) -> int:
-        return self.row
-
     def adjust(self, row: int = 0, col: int = 0) -> "Position":
         return type(self)(self.row + row, self.col + col)
 
 
+class Coordinate(NamedTuple):
+    """
+    Coordinate is (x, y) such that the left-most, bottom triangle is (0, 0).
```

Comment:
oh, is it that easy? I was under the impression that the other shows would need to use `(x, y)` coordinates with absolute distances (in, say, meters). I think we could implement that mapping; the metrics needed for that are the other things I was planning on storing in `Geometry`.